pom.xml

```diff
@@ -26,14 +26,15 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.26</version>
+    <version>3.51</version>
```

Comment:
3.52 resulted in the following error, presumably due to jenkinsci/plugin-pom#229:

```
[ERROR] Failed to execute goal org.codehaus.mojo:flatten-maven-plugin:1.1.0:flatten (flatten) on project throttle-concurrents: 8 problems were encountered while building the effective model for org.jenkins-ci.plugins:throttle-concurrents:2.0.2-SNAPSHOT
[ERROR] [ERROR] 'dependencies.dependency.version' for com.github.spotbugs:spotbugs-annotations:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for net.jcip:jcip-annotations:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for org.slf4j:slf4j-api:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for org.slf4j:log4j-over-slf4j:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for org.slf4j:jcl-over-slf4j:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for org.slf4j:slf4j-jdk14:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for org.kohsuke:access-modifier-annotation:jar is missing. @ 
[ERROR] [ERROR] 'dependencies.dependency.version' for commons-logging:commons-logging:jar is missing. @ 
```

Downgrading to 3.51 to work around the problem for now.